ci.sh

```diff
@@ -1,15 +1,37 @@
 #!/bin/bash
 
+set -o pipefail
 
+EXIT_STATUS=0
 if ! python -m autopep8 --exit-code -a -r --global-config .config-pep8 -i . ; then
   echo "Please run command 'python -m autopep8 -a -r --global-config .config-pep8 -i .' on your local and commit the result"
-  exit 1
+  EXIT_STATUS=1
 fi
 
 if ! python -m isort --check-only -rc . ; then
   echo "Please run command 'python -m isort -rc .' on your local and commit the result"
-  exit 1
+  EXIT_STATUS=1
 fi
 
-python -m pylint --rcfile .pylintrc appium test --errors-only
-python -m pytest test/unit/
+(
+  LINT_RESULT=$(python -m pylint --rcfile .pylintrc appium test --errors-only 2>&1 | tee /dev/tty)
+  if [[ $? -ne 0 ]] ; then
```

Comment:
I missed here, but I hope working for future as well.

https://github.com/PyCQA/pylint/blob/04cf3f8298fa1c718f950c90c34ee99a17f3c503/doc/user_guide/run.rst#L154